app/presenters/hyrax/menu_presenter.rb

```diff
@@ -0,0 +1,32 @@
+module Hyrax
+  # view-model for the admin menu
+  class MenuPresenter
+    def initialize(view_context)
+      @view_context = view_context
+    end
+
+    attr_reader :view_context
+
+    delegate :controller, :controller_name, :action_name, :content_tag,
+             :current_page?, :link_to, to: :view_context
+
+    # Returns true if the current controller happens to be one of the controllers that deals
+    # with workflow.  This is used to keep the parent section on the sidebar open.
+    def workflows_section?
+      controller.instance_of? Hyrax::Admin::WorkflowRolesController
+    end
+
+    def nav_link(href, **link_options)
+      content_tag(:li, class: ('active' if current_page?(href))) do
+        link_to(href, link_options) do
+          yield
+        end
+      end
+    end
+
+    # Draw a collaspable menu section. The passed block should contain <li> items.
+    def collapsable_section(text, **kwargs, &block)
+      CollapsableSectionPresenter.new(view_context, text, kwargs).render(&block)
```

Comment:
Shouldn't the `kwargs` be `**kwargs`?

I believe we should fully prefer keyword args (instead of mixed).
```ruby
CollapsableSectionPresenter.new(view_context: view_context, text: text, id: id, icon_class: icon_class, open: open)
```